starter-code/src/functions-and-arrays.js

```diff
@@ -60,6 +130,19 @@ var wordsFind = [
   'disobedience'
 ];
 
+function doesWordExist (array, word) {
+  if (wordsFind.length === 0) {
+    return undefined;
+  }
+  var result = false;
+  array.forEach(function(position) {
+    if (position === word) {
+      result = true;
+    }
+  });
+  return result;
+}
```

Comment:
Bien hecho. Varias cositas sobre el código. Podéis hacer que itere menos usando un bucle for y haciendo 'return true' cuando cumpla la condición. Y digo for y no forEach, porque el problema de map( ) y forEach( ) es que no dejan de iterar ante un break o un return. Cuando se encuentran con break o return, solo saltan esa iteración y pasan a la siguiente. Así podéis dejar de comprobar más nombres cuando ocurra la condición en vez de iterar sobre todos los nombres independientemente de que se cumpla la condición.